lib/r10k/source/base.rb

```diff
@@ -65,4 +65,17 @@ def accept(visitor)
       end
     end
   end
+
+  private
+
+  def derive_prefix(name,prefix)
```

Comment:
For the record, in Source::Git there's one place where prefix is checked, but it's dead code; https://github.com/puppetlabs/r10k/pull/341 will remove that.